### PR TITLE
feat(validation): Monitor validation

### PR DESCRIPTION
**Pull request by ewiseblatt**
@lwander 

I'm hacking in apt-get install python-dev as a dependency; I thought you added this earlier in the week but am not seeing it.

I'd like to have a way of setting --client_host (in the prometheus install) to expose the daemon to the network to be polled by a remote prometheus.

### CODE REVIEW

**Comment by lwander** [comment]
I'd added it to Halyard's installation of spinnaker-monitoring - this is the right place to do it though

**Comment by lwander** [comment]
I'd added it to Halyard's installation of spinnaker-monitoring - this is the right place to do it though

**Comment by lwander** [comment]
I'd added it to Halyard's installation of spinnaker-monitoring, maybe it's not up-to-date?

File: dev/validate_bom__config.py [hunk 1]
```diff
@@ -818,15 +819,33 @@ class MonitoringConfigurator(object):
...
+      raise ValueError('gateway is only applicable to '
+                       ' --monitoring_install_which="prometheus"')
 
   def add_config(self, options, script):
     """Implements interface."""
-    pass
+    if not options.monitoring_install_which:
+      return
+
+    script.append('sudo apt-get install python-dev -y')
```
Comment (lwander) [inline dev/validate_bom__config.py]: Shouldn't be necessary, I'm doing a github release for Halyard 0.26.0 that should cover this.

**Comment by ewiseblatt** [comment]
I ran this using master-latest-unvalidated last night with halyard-nightly
and saw that error until I added the pip install ahead of it.

**Pull request merged**